oak_functions/util/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "oak_functions_util"
```

Comment:
I like the idea of this crate! Is the intention that it will expose a shared API, but then offer different implementation of the same interface based on the underlying target? e.g. similar to the sys crate, or something like https://github.com/rust-lang/rust/blob/547369d3d881a9eb1de0d3e368f9a59aa7c648b0/library/std/src/os/unix/mod.rs#L38-L43 and per-os re-exported mods: https://github.com/rust-lang/rust/tree/master/library/std/src/os

Reply:
Then implementing something like https://github.com/project-oak/oak/issues/2630 would be almost trivial. And similarly for other functionality that may differ (or not exist) depending on the underlying target.

Reply:
Yes, in future the plan is that it will evolve in that direction.

Reply:
Yup. For now it might be something as simple as conditional compilation on different configurations.